starter-code/index.js

```diff
@@ -1,10 +1,89 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+let SortedList = function() {
+    this.items = [];
+    this.length = this.items.length;
+};
+
+SortedList.prototype.add = function(item) {
+    
+    this.items.push(item)
+    this.items.sort(function(i1, i2){
+        return i1-i2});
+    
+    this.length++;
+}
+SortedList.prototype.get = function(pos) {
+    if(pos > this.items.length){
+        return this.OutOfBounds();
+    }
+    return this.items[pos-1];
+}
+SortedList.prototype.max = function() {
+    if (this.items === 0) {
+        return this.EmptyList();
```

Comment:
Nunca os entrará en esa comprobación porque `this.items` nunca es estrictamente igual a `0`; ya que no estáis comparando la longitud, os falta el `.length`. Y como dije antes, `this.EmptyList()`, si no es una función constructora empieza por minúscula.